general/g.copy/g.copy.html

```diff
@@ -1,13 +1,48 @@
 <h2>DESCRIPTION</h2>
 
+The <em>g.copy</em> module creates a copy of existing raster maps,
+vector maps, or other elements. The copy is always created in
+the current mapset. The source data can be in the current mapset,
+in an explicitly specified mapset, or in a mapset which is in the
+current mapset search path (typically the PERMANENT mapset).
+
+<p>
+The maps and other elements to copy are specified in pairs
+<b>from</b>,<b>to</b> according to their types.
+Although typically only one map is copied in one module call,
+multiple pairs can be provided for each type and multiple types
+can be provided at the same time.
+
+<h3>Relation to mapsets</h3>
+
 A user may access data stored under the other mapsets listed in their
 mapset search path. However, the user may only modify data stored
 under their own current mapset. <em>g.copy</em> allows the user to copy
 existing data files <b>from</b> other mapsets <b>to</b> the user's
-current mapset ($MAPSET). The files to be copied must exist in the
+current mapset (<code>g.mapset -p</code>). The files to be copied must exist in the
 user's current mapset search path and location; output is sent to the
 relevant data element directory(ies) under the user's current mapset.
 
+<h3>Behavior on error</h3>
+
+When only one map or other element is requested to be copied and the copying
```

Comment:
```suggestion
Errors typically occur when a map or other element does not exist,
<b>from</b> and <b>to</b> are the same,  <b>to</b> element
already exists and overwriting (e.g., by <b>--overwrite</b>) is not
enabled or <b>to</b> element has an illegal name.
 
When only one map or other element is requested to be copied and the copying
```
Moved up from a paragraph further down.